app/schemas/tasks.py

```diff
@@ -32,6 +32,7 @@ class TranslationBatchResponse(BaseModel):
 
 class TTSRequest(BaseModel):
     text: str
+    return_audio_link: bool = False
```

Comment:
The response json contains the link but with the key `base64_string`, might want to change this to something like `audio_link`.

Reply:
What I meant, is that when someone requests the audio link, the response should contain the `audio_link` key, but if they don't, it should return `base64_string`, it seems like the latest change just makes it return `audio_link` in both cases.

Reply:
also, instead of force pushing, first pull the changes...because that causes a merge conflict in the  `requirements.txt` file.